ui/common/src/socket.ts

```diff
@@ -182,12 +180,10 @@ export default class StrongSocket {
     this.debug('send ' + message);
     if (!this.ws || this.ws.readyState === WebSocket.CONNECTING) {
       if (!noRetry) this.resendWhenOpen.push([t, msg.d, o]);
-    } else {
-      origSend.apply(this.ws, [message]);
-    }
+    } else this.ws.send(message);
   };
 
-  scheduleConnect = (delay: number): void => {
+  private scheduleConnect = (delay: number = this.options.pongTimeout): void => {
```

Comment:
pongTimeout is our nominal connect interval, so it's the default value for this call